README.md

```diff
@@ -7,7 +7,7 @@
 Describe the differences between a SQL and NoSQL database, and when you might use each.
 
 ```text
-Your answer...
+SQL databases are relational databases whereas NoSQL databases are non-relational databases.  Typically an SQL database is used with tables whereas a NoSQL database uses  collections and documents based.  NoSQL provides more flexibility because there is less structure.  An example of a NoSQL would be MongoDB.
 ```
 
```

Comment:
re: my initial comment: Oracle NoSQL is distinct from the category NoSQL. This has no direct bearing on the correctness of your answer, but the ambiguity potentially exists.